### DISCUSSION TITLE
Will be the next csharp copilot/ChatGpt-ready when launched?

### DISCUSSION THREAD

**Discussion by ali50m** [dotnet/csharplang#7239] [General]
Both copilot and ChatGpt are great tools helping to learn or utilize new feature. Will they be ready to provide the latest features or source code when the next dotnet/c# is launching?

Not sure if this question should be asked to dotnet/csharp/copilot/OpenAI, but it will be awesome we can get help from copilot/ChatGpt.

**Comment by Joe4evr** [comment]
This has nothing to do with the C# language.

**Reply by ali50m** [reply]
I guess copilot/OpenAI will need data to be trained.

**Reply by yaakov-h** [reply]
no more or less than humans also need to learn the new thing

**Reply by ali50m** [reply]
But machine learning can do them at the speed of light...  Plz consider making the AI ready when launched.

**Comment by huoyaoyuan** [comment]
> Both copilot and ChatGpt are great tools helping to learn or utilize new feature.

In fact they are the great tools to utilize **old** features. For example, dotnet/runtime#81796.
